hca-api-stub/voa-rest.js

```diff
@@ -8,14 +8,26 @@ function attach(app) {
     c7401: 'http://vaausesrapp803.aac.va.gov:7401/voa/voaSvc',
     e6401: 'https://vaausesrapp803.aac.va.gov:6401/voa/voaSvc',
     e8432: 'https://vaww.esrdev30.aac.va.gov:8432/voa/voaSvc',
+    preprod: 'https://vaww.esrpre-prod.aac.va.gov/voa/voaSvc',
+    prod: 'https://vaww.esr.aac.va.gov/voa/voaSvc'
   };
 
   const ds = loopback.createDataSource('soap',
     {
       connector: require('loopback-connector-soap'),
       remotingEnabled: true,
       wsdl: path.join(__dirname, './voa.wsdl'),
-      url: endpoint.e8432
+      url: endpoint.preprod,
+      security: {
+        scheme: 'ClientSSL',
+        certPath: path.join(__dirname, './healthcare.application.crt'),
+        keyPath: path.join(__dirname, './healthcare.application.key')
```

Comment:
Nope. I'll add logic to exclude the security object if there are no certs. Better solutions are invited.